### DISCUSSION TITLE
How to show related object properly?

### DISCUSSION THREAD

**Discussion by anton-petrov** [aminalaee/sqladmin#462] [Q&A] [answered]
Now I have a tables with FK(table Devices with foreign key DataSource).
So, please, explain, how to replace object reference (circled in red) with it's name or another field of DataSource?
<img width="789" alt="Screenshot 2023-04-13 103929" src="https://user-images.githubusercontent.com/2428315/231688628-a2d09d8b-29c3-456e-94d0-7a261b5a8232.png">
Thank you in advance :)

**Comment by aminalaee** [comment, marked as answer]
Hi,
You need to override the `__str__` method of your class.
